src/ts2fable.fs

```diff
@@ -11,28 +11,39 @@ open ts2fable.Naming
 open ts2fable.Read
 open ts2fable.Transform
 open ts2fable.Write
+open Yargs.Yargs
 
 let argv =
     yargs
-        .usage("Usage: ts2fable some.d.ts src/Some.fs")
+        .usage("Usage: ts2fable some.d.ts src/Some.fs  
+        ts2fable --multiple some.d.ts src/Some.fs")
         .command(U2.Case1 "$0 [files..]", "")
         .demandOption(U2.Case1 "files", "")
         .help()
+        .option("multiple",jsOptions<Options>(fun o ->
+            o.description <- Some "Import from multiple linked tsfiles"
+            o.boolean <- Some true
+        ))
         .argv
 
+
 let files = argv.["files"].Value :?> string array |> List.ofArray
 let tsfiles = files |> List.filter (fun s -> s.EndsWith ".ts")
 let fsfile = files |> List.tryFind (fun s -> s.EndsWith ".fs")
+let isMultiple = argv.["multiple"].Value :?> bool
 
-match tsfiles.Length, fsfile with
-| 0, _ -> failwithf "Please provide the path to a TypeScript file"
-| _, None -> failwithf "Please provide the path to the F# file to be written "
-| _, Some fsf ->
-    printfn "ts2fable %s" Version.version
-
-    // validate ts files exist
-    for ts in tsfiles do
-        if not <| fs.existsSync(!^ts) then
-            failwithf "TypeScript file not found: %s" ts
+// validate ts files exist
+for ts in tsfiles do
+    if not <| fs.existsSync(!^ts) then
+        failwithf "TypeScript file not found: %s" ts
 
+match tsfiles.Length, fsfile,isMultiple with
+| 0, _, _ -> failwithf "Please provide the path to a TypeScript file"
+| _, None, _ -> failwithf "Please provide the path to the F# file to be written "
+| _, Some fsf, false ->
+    printfn "ts2fable %s" Version.version
     writeFile tsfiles fsf
+| 1, Some _, true ->
+    printf "Not implemented"
```

Comment:
`Not implemented`
This code will be replaced ：
Resolving entry tsfile to multiple input tsfiles using `Typescript Compile API`